digdag-standards/src/main/java/io/digdag/standards/command/ecs/DefaultEcsClient.java

```diff
@@ -322,6 +322,45 @@ public void close()
         throw new RuntimeException("Failed to call EcsClient method");
     }
 
+    /**
+     * Retry function for {@link #getLog(String, String, Optional)}
+     * @param func
+     * @param GetLogEventsRequest
+     * @param <T>
+     * @return T
+     * @throws AmazonServiceException
+     */
+    @VisibleForTesting
+    <T> T retryForGetLog(Supplier<T> func, GetLogEventsRequest request) throws AmazonServiceException
+    {
+        for (int i = 0; i < rateLimitMaxRetry; i++) {
```

Comment:
Strictly speaking, this variable should be renamed or a new one should be created for this method.

Reply:
fixed https://github.com/treasure-data/digdag/pull/1626/commits/15c7b866769c5d326848405fc3b8c3387bb47b68